test/appframework_aws/s1/appframework_aws_test.go

```diff
@@ -1683,6 +1683,7 @@ var _ = Describe("s1appfw test", func() {
 			// ################## SETUP FOR STANDALONE ####################
 			// Download all test apps from S3
 			appVersion := "V1"
+			appListV1 := []string{appListV1[0]}
```

Comment:
So, for V1 we take 1 app only, but for V2 later on we take the normal appListV2?
Is that what we want to do?

Reply:
The test case involves updating the app as soon as we download it. If there are too many apps by the time we download the updated apps, the apps installed get updated and we are unable to check that previous install took place.

This added flakiness to the test. Changing the number of apps in this test case to counteract this issue